wiki/markdown/running-eiffel-intelligence-frontend.md

```diff
@@ -4,12 +4,14 @@
 
 Eiffel intelligence front-end is a Spring microservice distributed in a war file.
 
-To start the service, download the war file from
+Release Eiffel Interlligence front-end war files can be downloaded from Jitpack:
```

Comment:
Intelligence.
Also consider putting release between 'front-end' and 'war' so it reads like this:
Eiffel Intelligence front-end release war files can be downloaded from Jitpack
Otherwise, since release is a verb, when you start reading the sentence it reads like a request or command to do a release.

Reply:
Fixed.